hawk/app/models/primitive.rb

```diff
@@ -110,18 +108,15 @@ def shell_syntax
       end
 
       unless ops.empty?
-        cmd.push "params"
-
-        ops.each do |op, instances|
-          instances.each do |i, attrlist|
-            cmd.push "op #{op}"
-
-            attrlist.each do |key, value|
-              cmd.push [
-                key,
-                value.shellescape
-              ].join("=")
-            end
+        ops.each do |id, op|
```

Comment:
Lint/UnusedBlockArgument: Unused block argument - `id`. If it's necessary, use `_` or `_id` as an argument name to indicate that it won't be used.